### PR TITLE
[UTTT-4] -  build a tic tac toe board

### PR DESCRIPTION
**Pull request by adaviloper**

### CODE REVIEW

File: scenes/Cell.gd [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	var collision = get_node("CollisionShape2D").shape.get_rect()
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	pass
+
+
+func _on_input_event(viewport, event, shape_idx):
+	if event is InputEventMouseButton and event.button_index == MOUSE_BUTTON_LEFT and event.pressed:
+		print("assigning X")
```
Comment (jeromeportega) [inline scenes/Cell.gd:18]: You want to leave these debug statements in?

File: scenes/board.tscn [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	)
+	draw_line(
+		Vector2(startX + (width * 2 / 3), startY),
+		Vector2(startX + (width * 2 / 3), startY + width), 
+		Color.BLACK, lineWidth, 
+		true
+	)
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
```
Comment (jeromeportega) [inline scenes/board.tscn:55]: Do you need to define these built in methods, or did you just leave the boilerplate stubs in?

File: scenes/board.tscn [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+[node name="board" type="Node2D"]
+script = SubResource("GDScript_tuypk")
+
+[node name="Label" type="Label" parent="."]
+offset_left = 337.0
+offset_top = 100.0
+offset_right = 377.0
+offset_bottom = 123.0
+text = "Tic-Tac-Toe"
```
Comment (jeromeportega) [inline scenes/board.tscn:67]: `Ultimate-Tic-Tac-Toe`?

File: scenes/Cell.gd (hunk 1 above)
Comment (adaviloper) [reply scenes/Cell.gd:18]: Yeah, I'm choosing to just leaving it there for the moment until we work on the game manager part to handle the logic to toggle it based on the current player.

**Pull request merged**